src/couch/src/couch_file.erl

```diff
@@ -264,6 +265,19 @@ rename_file(Original) ->
         Else -> Else
     end.
 
+rename_dir(RootDelDir, Original, DbName) ->
```

Comment:
The situation where DbName can contains "/" was considered. For details, please see response to next comment.